shared/ui-components/src/page_components/listing/listing_sidebar/Apply.stories.tsx

```diff
@@ -27,7 +27,7 @@ storiesOf("Listing|Sidebar Apply", module).add("accepts postmarked applications"
   /* eslint-enable @typescript-eslint/ban-ts-ignore */
 })
 
-storiesOf("Listing|Sidebar Apply", module).add("shows multiple download URLs", () => {
+storiesOf("Listing Sidebar|Apply", module).add("shows multiple download URLs", () => {
```

Comment:
This story doesn't seem to render any differently for me than the baseline. Am I missing something subtle, or are we missing the data needed to power it correctly?

Reply:
@bk3c I believe it will look the same unless you click the download button to see multiple download links. I'd be fine with making this a default story and removing one of the other ones if that simplifies things.

Reply:
Ok, that was the subtle thing I was missing. No need to change the default, but maybe we can figure out a way to include instructions when we re-write the stories to CSF.